lib/internal/util/types.js

```diff
@@ -29,8 +29,48 @@ function isUint8Array(value) {
   return TypedArrayProto_toStringTag(value) === 'Uint8Array';
 }
 
+function isUint8ClampedArray(value) {
+  return TypedArrayProto_toStringTag(value) === 'Uint8ClampedArray';
```

Comment:
We should likely note in the documentation how these checks are performed because types can lie about what they are if the user overrides `Symbol.toStringTag`.

Reply:
This function is cached at Node.js startup and thus immune to userland overrides.

Reply:
Ah right right... completely zoned on the caching. Just verified locally that even preload script can't lie about this. Dismiss this comment then :-)

Reply:
Actually... @addaleax ... this would be something worth adding to the tests... specifically, ensuring that a user-provided class overriding a built-in set within a preload module cannot lie and that the checks will come up as false.

Reply:
>  a user-provided class overriding a built-in set within a preload module cannot lie 

I’m not sure what you mean by that. I have added a test that checks that overriding `Symbol.toStringTag` doesn’t fool this.